flatpak/org.nanuc.Axolotl.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://docs.flatpak.org/en/latest/freedesktop-quick-reference.html#appdata-files -->
+
+<component type="desktop">
+    <id>org.nanuc.Axolotl</id>
+    <name>Axolotl</name>
+    <summary>Axolotl is a crossplattform Signal client</summary>
+    <launchable type="desktop-id">org.nanuc.Axolotl.desktop</launchable>
+    <project_license>GPL-3.0</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+    <url type="homepage">https://axolotl.chat/</url>
+    <url type="bugtracker">https://github.com/nanu-c/axolotl/issues</url>
+    <content_rating type="oars-1.1" />
+    <description>
+        <p>
+            Axolotl is a complete Signal client, it allows you to create a Signal
+            account and have discussions with your contacts. Unlike the desktop
+            Signal client, Axolotl is completely autonomous and doesn't require
+            you to have created an account with the official Signal application.
+        </p>
+    </description>
+    <releases>
+        <release version="0.8.7" date="2020-09-27">
```

Comment:
So we have one more place to change the version string. next one should be 0.9.0